.dockerignore

```diff
@@ -0,0 +1,12 @@
+# Generated by Cargo
+**/target
+
+# Unneeded artifacts
+**/.DS_Store
+
+# The cache for docker container dependency
+.cargo
+.github
+
+# Environment
+.git
```

Comment:
we specify the context to be the binary in target, so not sure why exactly this is needed. see the build script for why this is both wrong and unnecessary

Reply:
> docker build target/release/polkadot -f chains/polkadot/Dockerfile -t analoglabs/connector-polkadot

the build context is `target/release/polkadot` in this case. ignoring `target/release/polkadot` is the last thing we want to do. that would presumably lead to an empty docker image